driver/src/test/java/org/neo4j/driver/ConfigTest.java

```diff
@@ -348,4 +361,73 @@ void shouldErrorWithInvalidUserAgent()
         assertThrows( IllegalArgumentException.class, () -> Config.builder().withUserAgent( null ).build() );
         assertThrows( IllegalArgumentException.class, () -> Config.builder().withUserAgent( "" ).build() );
     }
+
+    @Nested
+    class SerializationTest
+    {
+
+        @Test
+        void shouldSerialize() throws Exception
+        {
+            Config config = Config.builder().withMaxConnectionPoolSize( 123 ).withConnectionTimeout( 6543L, TimeUnit.MILLISECONDS ).withConnectionAcquisitionTimeout(
+                    5432L, TimeUnit.MILLISECONDS ).withConnectionLivenessCheckTimeout( 4321L, TimeUnit.MILLISECONDS ).withMaxTransactionRetryTime( 3210L, TimeUnit.MILLISECONDS ).withFetchSize( 9876L ).withEventLoopThreads(
+                    4 ).withoutEncryption().withTrustStrategy( Config.TrustStrategy.trustAllCertificates() ).withUserAgent( "user-agent" ).withDriverMetrics().withRoutingTablePurgeDelay( 50000, TimeUnit.MILLISECONDS ).withLeakedSessionsLogging().build();
```

Comment:
It might be worth formatting this and maybe keeping every option on a new line to improve readability.

Reply:
Autoformat is an a** with those builders…